### PR TITLE
Temporal bug fix: typo module

### PR DESCRIPTION
**Pull request by doorooful**
- Handle exception when unexpected response gotten from external API.

### CODE REVIEW

**Comment by doorooful** [comment]
Test result:
Screenshot 2023-09-22 at 3 54 56 PM

File: dt-demo-be/modules/typo/run.py [hunk 1]
```diff
@@ -29,11 +29,24 @@ def run(fileName):
...
+        try:
+            '''
+            Here loads typo response,
+            and match typo words to the expected(correct) words
+            '''
+            data_dict = json.loads(data)
+            err_info = data_dict['errInfo']
+            for err in err_info:
+                typo_detected.append("typo: "+err["orgStr"]+", "+"expected word: "+err["candWord"])
+        except json.JSONDecodeError as e:
```
Comment (yoouung) [inline dt-demo-be/modules/typo/run.py:46]: can you explain why you used `except` instead of `catch`?

Comment (coalalalala) [reply dt-demo-be/modules/typo/run.py:46]: It's just about a python syntax

Comment (yoouung) [reply dt-demo-be/modules/typo/run.py:46]: 😂

**Review by yoouung — APPROVED** [review]

**Pull request merged**